database.js

```diff
@@ -46,6 +46,13 @@ async function queryAllEvents() {
   return events;
 }
 
+async function queryAllEventNames() {
+  let conn = await pool.getConnection();
+  let events = await conn.query("SELECT eventID FROM LICCB.events");
+  conn.release();
```

Comment:
This is the old way of querying the database so all occurrences of this will need to be addressed.  Please see the changes in #58 or other queries in this branch to see how to use Sequelize for raw queries.  It should be pretty straight forward, message me on slack if you need help.

Reply:
Thank you for reminding me. I updated the functions to use sequelize